Sources/AppCoinsSDK/Domain/Entities/Currency.swift

```diff
@@ -12,9 +12,17 @@ internal class Currency: Codable {
     let currency: String
     let label: String
     let sign: String
-    let type: String
+    let type: String?
     let flag: URL?
     
+    init(userCurrencyRaw: ConvertCurrencyRaw) {
```

Comment:
Replace `userCurrencyRaw` with `convertCurrencyRaw` here. This change will provide the flexibility to use it for other scenarios involving the `convertCurrency` request. By doing this, we avoid limiting ourselves to just one use case.